src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/ResolveOverlappingTrainingDateRequest/ResolveOverlappingTrainingDateRequestCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MediatR;
+using SFA.DAS.CommitmentsV2.Types;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.CommitmentsV2.Application.Commands.ResolveOverlappingTrainingDateRequest
+{
+    public class ResolveOverlappingTrainingDateRequestCommand : IRequest
+    {
+        public long? ApprenticeshipId { get; set; }
+        public long? DraftApprenticeshipId { get; set; }
```

Comment:
No need for draftapprenticeshipId, as the previous employer is resolving the request. Just pass through apprenticeshipId

Reply:
Done